src/pages/UserProfile/components/Account/Account.tsx

```diff
@@ -7,13 +7,14 @@ import { AccountButton, FlexBoxColumn, UAvatar, UCard } from './Account.styles';
 
 interface AccountProps {
 	username: string;
+	userid: string;
 }
 
-export const Account = ({ username }: AccountProps): JSX.Element => (
+export const Account = ({ username, userid }: AccountProps): JSX.Element => (
 	<UCard>
 		<CardContent>
 			<FlexBoxColumn>
-				<UAvatar {...stringAvatar(username)} />
+				<UAvatar {...stringAvatar(username, userid)} />
```

Comment:
When you move `Avatar` to separate component, and then you can set field style and add `backgroundImage: 'url(${GetUserIconUrl(parseInt(id))})'`, there. It will be much better